pkg/daemon/daemon.go

```diff
@@ -129,7 +129,7 @@ func New(
 	}
 	InitializeOffsetMaps()
 	pluginManager := registerPlugins(plugins)
-	eventChannel := make(chan event.EventChannel, 10)
+	eventChannel := make(chan event.EventChannel, 50)
```

Comment:
Why the change here?

Reply:
Just to catch any failures , Increased the buffere size of channel so , wont block until 50 events if happens to be in the queue